flask_meld/message.py

```diff
@@ -0,0 +1,60 @@
+from .component import get_component_class
+
+
+def process_message(message):
+    meld_id = message["id"]
+    action = message["action"]
+    component_name = message["componentName"]
+    meld_id = meld_id
+
+    Component = get_component_class(component_name)
+    component = Component(meld_id)
+    payload = action["payload"]
+
+    if "syncInput" in action["type"]:
+        if hasattr(component, payload["name"]):
+            setattr(component, payload["name"], payload["value"])
+
+    elif "callMethod" in action["type"]:
+        data = message["data"]
+        call_method_name = payload.get("name", "")
+        method_name = call_method_name
+
+        for arg in component.__attributes__():
+            try:
+                value = data.get(arg)
+                setattr(component, arg, value)
+
+            except ValueError:
+                pass
+            except AttributeError as e:
+                print(f"{e}: {arg}-{value}")
+
+        method_name = call_method_name
+        params = None
+
+        if "(" in call_method_name and call_method_name.endswith(")"):
+            param_idx = call_method_name.index("(")
+            params_str = call_method_name[param_idx:]
+
+            # Remove the arguments from the method name
+            method_name = call_method_name.replace(params_str, "")
+
+            # Remove parenthesis
+            params_str = params_str[1:-1]
+            if params_str == "":
+                return method_name
+            else:
```

Comment:
no `else` needed if you return (= one indent less (_flat is better than nested_))